### DISCUSSION TITLE
.Net MAUI Tesseract and Android

### DISCUSSION THREAD

**Discussion by CyborgDE** [emgucv/emgucv#809] [Q&A] [answered]
Hello,

I'm having trouble getting it to work, does anyone have experience with it?

Best regards
CyborgDE

**Comment by emgucv** [comment, marked as answer]
With the 4.6.0 professioal licesne, you can download this file: 

> libemgucv-pro-4.6.0.5131.zip

Unzip the file, open "Solution\CrossPlatform\Emgu.CV.CrossPlatform.sln" solution with VS2022. 

You may need to got to "Emgu.CV.Example\MAUI\MauiDemoApp" folder, delete "obj" folder, and run the following two commands to restore the worload and nuget packages correctly:
```
dotnet workload restore
dotnet restore
```
Visual Studio 2022 doesn't seem to correctly restore nuget package for MAUI project ... Running the scripts above to restore workload and nuget package is required to compile the solution when we are using VS 17.4.4.

Once that is done, you can compile the "MauiDemoApp" in Visual Studio:
![image](https://user-images.githubusercontent.com/2035816/212973296-3f57fcc2-2bdc-4fed-9934-71ed5c3b4526.png)

After which you can deploy and run the app on Android emulator or device:
![image](https://user-images.githubusercontent.com/2035816/212974376-5f535829-8e6c-46fa-af0f-9a763d6fc769.png)

Choose the "Tesseract OCR" demo, and you will be able to see tesseract ocr in action under android:
![image](https://user-images.githubusercontent.com/2035816/212974602-a5c0d62f-8bfe-4418-8095-ff2b56818bdd.png)
